provider/clusterrosa/hcp/resource.go

```diff
@@ -320,6 +320,14 @@ func (r *ClusterRosaHcpResource) Schema(ctx context.Context, req resource.Schema
 				Description: "Wait until the cluster standard compute pools are created. The waiter has a timeout of 60 minutes, with the default value set to false. This can only be provided when also waiting for create completion.",
 				Optional:    true,
 			},
+			"cluster_wait_timeout_in_minutes": schema.Int64Attribute{
+				Description: "This value sets the maximum duration in minutes to wait for the cluster to be in a ready state.",
+				Optional:    true,
+			},
+			"resource_wait_timeout_in_minutes": schema.Int64Attribute{
+				Description: "This value sets the maximum duration in minutes to wait for the cluster resource to be in a ready state.",
```

Comment:
I think this might be confusing for customers, as the cluster resource in terraform lingo means the cluster itself
Maybe subresources? But I still feel like we should highlight that it is for the default machine pool somehow as it might still be confused with other sub resources such as idp etc

Reply:
I'll change it back to `Default_wait_timeout_in_minutes` as I'll be using that value for classic clusters also